Userland/Libraries/LibJS/AST.h

```diff
@@ -89,8 +89,8 @@ class ASTNode : public RefCounted<ASTNode> {
     explicit ASTNode(SourceRange);
 
 private:
-    RefPtr<SourceCode> m_source_code;
     u32 m_start_offset { 0 };
+    RefPtr<SourceCode> m_source_code;
     u32 m_end_offset { 0 };
```

Comment:
IMO this deserves a comment on the order of member variables here, especially since the savings in the PR depend on the padding hole this creates.

Reply:
Absolutely!